hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/master/MasterStatusTmpl.jamon

```diff
@@ -644,17 +641,20 @@ AssignmentManager assignmentManager = master.getAssignmentManager();
         </%if>
         <td><% tableState.getState() %></td>
         <td><% openRegionsCount %></td>
-        <%if (openingRegionsCount > 0) %> <td><a href="/rits.jsp?table=<% tableName.getNameAsString() %>&state=OPENING"><% openingRegionsCount %></td> <%else><td><% openingRegionsCount %></td> </%if>
-        <td><% closedRegionsCount %></td>
-        <%if (closingRegionsCount > 0) %> <td><a href="/rits.jsp?table=<% tableName.getNameAsString() %>&state=CLOSING"><% closingRegionsCount %></td> <%else><td><% closingRegionsCount %></td> </%if>
-        <td><% offlineRegionsCount %></td>
-        <td><% failedRegionsCount %></td>
-        <td><% splitRegionsCount %></td>
+        <%if (openingRegionsCount > 0) %> <td><a href="/table.jsp?name=hbase%3Ameta&scan_table=<% tableName.getNameAsString() %>&scan_region_state=OPENING"><% openingRegionsCount %></td> <%else><td><% openingRegionsCount %></td> </%if>
```

Comment:
Should we URLEncode the values produced by `tableName.getNameAsString` ?

Reply:
Let me do it.